schemas/storedPassword.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
What makes the difference between `StoredPassword` and `BadpwFailedLogins` where the former benefits from a json schema where the latter doesn't require it?